SEIR/test/data/config.yml

```diff
@@ -0,0 +1,190 @@
+## This config file will not generate a makefile as currently configured.
```

Comment:
I don't think we want another config.yml to maintain.  Can we use either COVID19_Minimal Repo or the config.yml in COVIDScenarioPipeline?